src/lib/dfp/render-advert.ts

```diff
@@ -146,6 +146,38 @@ const addContentClass = (adSlotNode: HTMLElement) => {
 	}
 };
 
+/* Centre certain slots in their containers, this is class is added dynamically to avoid rendering quirks with the ad label, and variable width ads. */
+const addContainerClass = (adSlotNode: HTMLElement, isRendered: boolean) => {
+	const centreAdSlots = [
+		'dfp-ad--top-above-nav',
+		'dfp-ad--merchandising-high',
+		'dfp-ad--merchandising',
+	];
+	return fastdom
+		.measure(() => {
+			if (
+				isRendered &&
+				!adSlotNode.classList.contains('ad-slot--fluid') &&
+				adSlotNode.parentElement?.classList.contains(
+					'ad-slot-container',
+				) &&
+				centreAdSlots.includes(adSlotNode.id)
+			) {
+				return true;
+			}
+			return false;
```

Comment:
The if statement is redundant here?
```suggestion
			return (
				isRendered &&
				!adSlotNode.classList.contains('ad-slot--fluid') &&
				adSlotNode.parentElement?.classList.contains(
					'ad-slot-container',
				) &&
				centreAdSlots.includes(adSlotNode.id)
			);
```

Reply:
Nice spot, thanks 👍